mmv1/templates/terraform/pre_create/integrations_auth_config.go.erb

```diff
@@ -0,0 +1,17 @@
+// Move client certificate to url param from request body
```

Comment:
Do you have any idea why this isn't documented? https://cloud.google.com/application-integration/docs/reference/rest/v1/projects.locations.authConfigs/create

It does appear in API Explorer correctly, though.